### PR TITLE
feat: remove legacy cry light from TrackView

### PR DESCRIPTION
**Pull request by pollend**
## What does this PR do?

I don't think Cry Light is used in the current engine.

### CODE REVIEW

File: Code/Legacy/CryCommon/IMovieSystem.h [hunk 1]
```diff
@@ -880,7 +880,7 @@ struct IAnimSequence
         eSeqFlags_NoSpeed             = BIT(13), //!< Cannot modify sequence speed - TODO: add interface control if required
      // eSeqFlags_CanWarpInFixedTime  = BIT(14), //!< @deprecated - Timewarp by scaling a fixed time step - removed July 2017, unused
         eSeqFlags_EarlyMovieUpdate    = BIT(15), //!< Turn the 'sys_earlyMovieUpdate' on during the sequence.
-        eSeqFlags_LightAnimationSet   = BIT(16), //!< A special unique sequence for light animations
+     // eSeqFlags_LightAnimationSet   = BIT(16), //!< A special unique sequence for light animations
```
Comment (pollend) [inline Code/Legacy/CryCommon/IMovieSystem.h:883]: any historical information about this bitfield?

**Comment by guillaume-haerinck** [comment]
Have you tried to use a 03de light in the track view and animate it ? As maybe some of this old logic, even if unused, still need to be ported over to the new lights rather than being deleted

**Comment by pollend** [comment]
> Have you tried to use a 03de light in the track view and animate it ? As maybe some of this old logic, even if unused, still need to be ported over to the new lights rather than being deleted

you do this through the behavior system.

**Review by gadams3 — APPROVED** [review]

**Review by nick-l-o3de — APPROVED** [review]

**Review by lsemp3d — APPROVED** [review]

**Comment by nick-l-o3de** [comment]
This needs to be rebased on latest, and this issue resolved
```
Gems/LyShine/Code/Editor/Animation/UiAnimViewNodes.cpp(1079,109): error C2220: the following warning is treated as an error:
     int CUiAnimViewNodesCtrl::ShowPopupMenuSingleSelection(UiAnimContextMenu& contextMenu, CUiAnimViewSequence* pSequence, CUiAnimViewNode* pNode)
Gems/LyShine/Code/Editor/Animation/UiAnimViewNodes.cpp(1079,109): warning C4100: 'pSequence': unreferenced formal parameter
```

**Comment by nick-l-o3de** [comment]
Appears to have conflicts, plus 
```
 D:/workspace/o3de/Gems/LyShine/Code/Editor/Animation/UiAnimViewNodes.cpp(1078,109): error C2220: the following warning is treated as an error [D:\workspace\o3de\build\windows\External\LyShine-9ec8fc3a\Code\LyShine.Tools.Static.vcxproj]

[2024-06-25T17:51:53.430Z]   int CUiAnimViewNodesCtrl::ShowPopupMenuSingleSelection(UiAnimContextMenu& contextMenu, CUiAnimViewSequence* pSequence, CUiAnimViewNode* pNode)

[2024-06-25T17:51:53.430Z]                                                                                                               ^

[2024-06-25T17:51:53.430Z] D:/workspace/o3de/Gems/LyShine/Code/Editor/Animation/UiAnimViewNodes.cpp(1078,109): warning C4100: 'pSequence': unreferenced formal parameter [D:\workspace\o3de\build\windows\External\LyShine-9ec8fc3a\Code\LyShine.Tools.Static.vcxproj]

```

**Review by AMZN-daimini — APPROVED** [review]

**Comment by nick-l-o3de** [comment]
Failed with
```
/data/workspace/o3de/Code/Editor/TrackView/TrackViewNodes.cpp:2491:24: error: no member named 'Light' in 'AnimNodeType'
```
Might be due to the other stuff?

**Pull request closed without merging**